### PR TITLE
feat: Add more params for Williams Fractal

### PR DESCRIPTION
**Pull request by LeeDongGeon1996**
### Description

 - Add more params for Williams Fractal

Fixes partially #82 

### Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code and included any verifying manual calculations
- [x] I have added or updated unit tests that prove my fix is effective or that my feature works, and achieves sufficient code coverage.  New and existing unit tests pass locally and in the build (below) with my changes
- [x] My changes generate no new warnings and running code analysis does not produce any issues
- [x] I have added or run the performance tests that depict optimal execution times
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: stock_indicators/indicators/fractal.py [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-# def get_fractal(quotes: Iterable[Quote], left_span: int, right_span: int = None):
-#     fractal_results = CsIndicator.GetFractal[Quote](CsList(Quote, quotes), left_span, right_span)
-#     return FractalResults(fractal_results, FractalResult)
 
-def get_fractal(quotes: Iterable[Quote], window_span: int = 2):
+@overload
+def get_fractal(quotes: Iterable[Quote], window_span: int = 2, end_type = EndType.HIGH_LOW) -> "FractalResults[FractalResult]": ...
+@overload
+def get_fractal(quotes: Iterable[Quote], left_span: int, right_span: int, end_type = EndType.HIGH_LOW) -> "FractalResults[FractalResult]": ...
+def get_fractal(quotes, left_span = None, right_span = EndType.HIGH_LOW, end_type = EndType.HIGH_LOW):
```
Comment (DaveSkender) [inline stock_indicators/indicators/fractal.py:17]: `right_span` value here is not an `enum`

**Review by DaveSkender — APPROVED** [review]
Just one minor thing to fix to get the tests working correctly.

File: stock_indicators/indicators/fractal.py (hunk 1 above)
Comment (LeeDongGeon1996) [reply stock_indicators/indicators/fractal.py:17]: Thanks, it was enum problem.

Comment (DaveSkender) [reply stock_indicators/indicators/fractal.py:17]: I was looking at `right_span = EndType.HIGH_LOW` specifically.  right + left span are user-supplied integer values and would never be an `EndType` enum value.

Comment (LeeDongGeon1996) [reply stock_indicators/indicators/fractal.py:17]: Ah, right. but it's just for the role as a third parameter of the first overloaded method.

**Pull request merged**